src/bundle/Resources/public/js/CKEditor/custom-tags/block-custom-tag/custom-tag-editing.js

```diff
@@ -48,7 +48,9 @@ class IbexaCustomTagEditing extends Plugin {
                     </button>`;
 
                     domElement.innerHTML = `
-                        <div class="ibexa-custom-tag__header-title">${customTagConfig.label}</div>
+                        <div class="ibexa-custom-tag__header-title" data-cke-tooltip-text="${customTagConfig.label}">${
+                        customTagConfig.label
+                    }</div>
```

Comment:
```suggestion
                        <div class="ibexa-custom-tag__header-title" data-cke-tooltip-text="${customTagConfig.label}">
                            ${customTagConfig.label}
                        </div>
```

This will be more readable in my opinion